File: gen.py
import random;

n = 5
q = 5
print(n)
for i in range(n):
    print(random.randint(1,10), end=" ")

print(f"\n{q}")
for i in range(q):
    print(f"{random.randint(1,n-1)} {random.randint(1,10)}")

File: Gen.java
import java.util.*;
import java.io.*;

public class Gen {
    static int rand(long a,long b){
        return (int)(Math.random()*(b-a)+a);
    }
    static public String randString(int n) {
        int leftLimit = 97; // letter 'a'
        int rightLimit = 122; // letter 'z'
        int targetStringLength = n;
        Random random = new Random();

        String generatedString = random.ints(leftLimit, rightLimit + 1)
          .limit(targetStringLength)
          .collect(StringBuilder::new, StringBuilder::appendCodePoint, StringBuilder::append)
          .toString();

        return generatedString;
    }   
    public static void main(String[] args) throws IOException {
        int n = rand(1,10);
        System.out.println( randString(n));
        System.out.println( randString(n));
    }
}
